js/ui/calendar/ui.calendar.js

```diff
@@ -58,7 +58,7 @@ var Calendar = Editor.inherit({
         return extend(this.callBase(), {
 
             /**
-            * @name dxCalendarOptions.hoverStateEnabled
+            * @name dxCalendarOptions.hoverStateEnabled22
```

Comment:
What?

Reply:
Yep, it turned out we have problems with the new check run on the Docker image

Reply:
And this change was made for testing and was expected to be rolled back yesterday

Reply:
I'm ready with it.